### ISSUE TITLE
Each page should have a 'Skip to Content' link for screen readers

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1442]
Adding a 'skip to content' link for screen readers will reduce the amount of time and frustration a user has while waiting for all of the navigation links to be read on each page.

Update the main `application.html.erb` file by adding a 'skip to content' link that will bring users to the main content of the page. The link should only be visible to screen readers.

refer to these documents for instructions on how to implement: https://webaim.org/techniques/skipnav/ and https://accessible360.com/accessible360-blog/use-aria-label-screen-reader-text/

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'High' priority.
